depthai_ros_driver/include/depthai_ros_driver/device_ros.hpp

```diff
@@ -247,27 +249,28 @@ class DeviceROS : public dai::DeviceBase {
                 case dai::DatatypeEnum::Buffer:
                     break;
                 case dai::DatatypeEnum::CameraControl:
-                    // _pub_t[name] = std::thread{generate_pub_lambda<depthai_datatype_msgs::RawCameraControl>(_pub_nh,
+                    // _pub_t[name] = std::thread{generate_pub_lambda<depthai_datatype_msgs::RawCameraControl>(nh,
                     // name, 10)};
                     break;
                 case dai::DatatypeEnum::IMUData:
                     // @TODO: support IMU
-                    // _pub_t[name] = std::thread{generate_pub_lambda<depthai_datatype_msgs::RawIMUData>(_pub_nh, name,
+                    // _pub_t[name] = std::thread{generate_pub_lambda<depthai_datatype_msgs::RawIMUData>(nh, name,
                     // 10)}; // Not supported
                     break;
                 case dai::DatatypeEnum::ImageManipConfig:
                     break;
                 case dai::DatatypeEnum::ImgDetections:
-                    _pub_t[name] = std::thread{
-                            generate_pub_lambda<depthai_datatype_msgs::RawImgDetections>(_pub_nh, name, 10)};
-                    break;
-                case dai::DatatypeEnum::ImgFrame:
                     _pub_t[name] =
-                            std::thread{generate_pub_lambda<depthai_datatype_msgs::RawImgFrame>(_pub_nh, name, 10)};
+                            std::thread{generate_pub_lambda<depthai_datatype_msgs::RawImgDetections>(nh, name, 10)};
                     break;
+                case dai::DatatypeEnum::ImgFrame: {
+                    _pub_t[name] = std::thread{generate_pub_lambda<depthai_datatype_msgs::RawImgFrame>(nh, name, 10)};
+                    boost::function<bool(std_srvs::Trigger::Request&, std_srvs::Trigger::Response&)> fn =
+                            boost::bind(&DeviceROS::_defaultCameraInfo, this, name, ph::_1, ph::_2);
+                    _camera_info_default_srvs[name] = nh.advertiseService("reset_camera_info", fn);
```

Comment:
Is this service for resetting RoI control?

Reply:
No, this is the service for setting the camera info to default